xml_inscriptions/transcribed/CA.Malibu.JPGM.L.80.AD.10.2.xml

```diff
@@ -153,14 +168,24 @@
                   <name type="cognomen" key="Pantagathus">
                      <expan><abbr>Pantag</abbr><ex>athi</ex></expan>
                   </name>
-               </persName>, <expan><abbr>S<unclear>u</unclear></abbr><ex>ulpicianum</ex></expan>.
+               </persName>, <expan><abbr>S<unclear>u</unclear></abbr><ex>lpicianum sc. opus</ex></expan>.
```

Comment:
I've corrected this already, but for reference: if you want to have "sc." or "scil." (scilicet, supplying something understood but not expressed), the markup is `<supplied reason="subaudible">`; it's not technically part of the abbreviation.